### ISSUE TITLE
a11y: Insufficient color contrast on Google Login button

### ISSUE THREAD

**Issue by Danielle254** [guardiansofthegit-collab/elecretanta#510]
![Image](https://github.com/user-attachments/assets/e7316c2c-da6d-4f48-88e2-ba11699a2c1f)

Appears twice on the homepage `('/')`
Model the fix from [PR 356](https://github.com/LetsGetTechnical/elecretanta/pull/356)
